### DISCUSSION TITLE
install medialibrary-pro or laravel-comments in a github action

### DISCUSSION THREAD

**Discussion by fredattack** [spatie/laravel-medialibrary#3102] [General] [answered]
Hi guys, 

We have paid for your wonderful packages ( medialibrary-pro and laravel comments) and we would like to create an automated github action to test the application.

The installation of the packages fails because the credentials for these packages is not mentioned. 
How can I pass the license without compromising my credentials? 

From nivelles in 🇧🇪 ... 😃

#comeOnBelgium

**Comment by freekmurze** [comment, marked as answer]
Do not solve this by making a public fork of our repo or register it on Packagist. This is forbidden by our license.

To use our paid packages in your CI, you must define a GitHub secret `COMPOSER_AUTH` and use the contents of your `auth.json` as the value. 

Here's an example: https://gist.github.com/pixelbrackets/78504af8aef55439113d521bb6d22d4f#github
